README.md

```diff
@@ -1,20 +1,11 @@
-# Test Observability Action
+# Test Observability Action Src
 
-An action to push junit files to Ably's test observability server.
+Source code for https://github.com/ably/test-observability-action
 
-# Inputs
+Changes to the source are made here and then deployed to the downstream
```

Comment:
With markdown generally you shouldn't add wrapping yourself; you just let the markdown renderer take care of that for you since it can wrap dynamically based on what size it's rendering the document.

Reply:
Same applies to line 10/11